README.md

```diff
@@ -1,8 +1,15 @@
 # Band Oracle Contract
-The Cadence smart contract(s) enabling Flow native integration of the Band Protocol oracle network. The oracle network will provide price updates for crypto, fiat and commodities financial data for consumption by Flow applications. 
+The Cadence smart contract enabling integration of the Band Protocol Oracle network. The oracle network offers ~100 cryptocurrency [price quotes](https://data.bandprotocol.com/) available to any Cadence applications, contracts or transactions.  
 
 To learn more about Band Protocol please refer to: https://faq.bandprotocol.com/
 
+## Contract Addresses 
+
+|Name|Testnet|Mainnet|
+|----|-------|-------|
+|[BandOracle](contracts/BandOracle.cdc)|[0x4772ee0aba864de9](https://flow-view-source.com/testnet/account/0x4772ee0aba864de9/contract/BandOracle)|[]()|
```

Comment:
Could we direct to ContractBrowser instead here? https://contractbrowser.com/A.4772ee0aba864de9.BandOracle